flutter/lib/common/widgets/peer_tab_page.dart

```diff
@@ -824,13 +824,14 @@ class _PeerViewDropdownState extends State<PeerViewDropdown> {
         setState(() {
           menuPos = RelativeRect.fromLTRB(x, y, x, y);
         });
+        showMenu(
+          context: context,
+          position: menuPos,
+          items: items,
+          elevation: 8,
+        );
       },
-      onTap: () => showMenu(
-        context: context,
-        position: menuPos,
-        items: items,
-        elevation: 8,
-      ),
+      onTap: () => {}
```

Comment:
This part of coce looks stupid.

Reply:
why setState here?

Reply:
Why is different from `sort by`?

Reply:
Why do not share the code with `sort by`?

Reply:
Ok, sorry. I used setState because the menu position was not updating. But now I have removed it and made it work same as `Sort by`.